FHEM/00_SIGNALduino.pm

```diff
@@ -3827,8 +3848,27 @@ sub SetSens {
 }
 
 
+############################################################################
+########## Section & cc1101 settings for manufacturer specific´s  ##########
+package cc1101_manspecific;
```

Comment:
Das bringt es ja jetzt irgendwie nicht so ganz, das Package in 00_SIGNALDuino zu stecken.
So kann das ja immer noch keiner für seine Zwecke einbinden und nutzen.

Vermutlich ist auch nur der Zeil
'address' aus dem array ein cc1101 generischer Teil, wobei sich mir auch nicht erschließt, welchen mode der darstellen soll. Vermutlich ist das zum Adressieren und dann wäre es vielleicht die bessere Variante gewesen kein array zu nehmen sondern einen hash in dem man ein Mapping machen kann?
Vielleicht ist hier aber auch die Reihenfolge wichtig, könnte ja beim Registerschreiben durchaus sein,

Worin besteht das Problem, wenn wir die Register zum `PCA301`  schreiben?
https://github.com/RFD-FHEM/RFFHEM/blob/1000a4edd764d21381c453a592aa6e8f17c2e558/FHEM/lib/SD_ProtocolData.pm#L2693

Reply:
1) Wenn die Register dort verankert werden, so kann der User nicht umstellen auf den Modus wenn die Datei fehlen würde.

2) Das einsortieren der Register in die Protokolldatei könnte unter umständen dazu führen, das mehrere Sensoren die gleich Register aufsich bringen. (Das wissen wir natürlich nicht und was in Zukunft kommt, keine Ahnung)

3) muss ich mir anschauen ;-) ob es in der Umsetzung dann noch klappt.

Reply:
Zu 1. Wenn die Datei fehlt, kann man das Modul ohnehin nicht wirklich nutzen und der Fall kann nur vorkommen, wenn es jemand löscht. Also eher unwahrscheinlich oder?

2.vom Prinzip ist das ja heute schon so. Fast alle Protokolle sind für die Registersettings default erstellt.
Sobald wir den Packethandler vom cc1101 verwenden, brauchen wir nur noch den passenden Eintrag aus der Protokolliste hatte ich gedacht.
Der Universalempfänger ist doch ab diesem Zeitpunkt ein Protokoll spezifischer Empfänger. Das ist zumindest mein Verständnis der Zusammenhänge, kann auch falsch sein.